src/main.js

```diff
@@ -0,0 +1,211 @@
+// factory function to create new board object
+var Board = function (pegs, discs) {
```

Comment:
I would recommend using post es-6 syntax using let and const to declare variables